tests/unit/store/flask.spec.js

```diff
@@ -422,14 +428,14 @@ describe("store/flask", () => {
         const bound_ping_system_status = ping_system_status.bind(context);
         await bound_ping_system_status();
 
-        expect(store.state.flask.status_uuid).toEqual(
+        expect(store.state.flask.status_uuid).toStrictEqual(
           STATUS.MESSAGE.CALIBRATION_NEEDED
         );
 
-        expect(store.state.playback.playback_state).toEqual(
+        expect(store.state.playback.playback_state).toStrictEqual(
           playback_module.ENUMS.PLAYBACK_STATES.CALIBRATION_NEEDED
         );
-        expect(store.state.playback.barcode).toEqual(null);
+        expect(store.state.playback.barcode).toBeNull();
```

Comment:
I think that this `expect` statement would be true even at the beginning of the test.  Please confirm.  You may need to commit something else to the store at the beginning of the test to ensure that you are testing that the value actually changes as a result of the function being tested

Reply:
Sorry, fixing the testcases its not the part of the task.
(a) Eslint errors required corrections.
(b) getters removal only.

Reply:
I thought this would be a very small and quick correction, but if you feel it would take too much time during the current task, then please put it on an Asana card in the backlog so that we do not lose track of it for the future.